sdk/storage/src/table/query_result.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{ContinuationToken, TableEntity};
+use azure_core::errors::AzureError;
+use http::HeaderMap;
+use hyper::body;
+use serde::{de::DeserializeOwned, Serialize};
+
+pub struct QueryResult<T>
```

Comment:
I wonder if `QueryResult` is the right name. This doesn't give any indication that this response contains a continuation token. Additionally, `Result` in the name, makes me think that this would be a top of `std::result::Result`, which it is not.

Reply:
You are right. Do you think `QueryResponse` would be better?

Reply:
Does `PaginatedResponse` make sense?

Reply:
Done!